.github/workflows/pytest-coverage.yaml

```diff
@@ -0,0 +1,52 @@
+on:
+  pull_request:
+  push:
+    branches: [main, master]
+
+name: pytest-coverage
```

Comment:
The name `pytest-coverage` here is intended to match the `test-coverage` action for R, even though we don't really make use of the coverage stats in this workflow.